### PR TITLE
add(templates): OhMySMTP

### PR DESCRIPTION
**Pull request by paul-oms**
| Name                 | Link                 |
| -------------------- | -------------------- |
| OhMySMTP Templates|https://github.com/ohmysmtp/templates|

A set of Transactional HTML Email Templates, built with Tailwind CSS and Maizzle

---

- [x] My item is in the right category
- [x] My item is logically grouped below similar items
- [x] My item's name and description respects the conventions of the list
- [x] My item is awesome
- [x] My item is in line with the [Tailwind brand usage guidelines](https://tailwindcss.com/brand#usage)
- [x] I have read and followed the [contribution guidelines](https://github.com/aniftyco/awesome-tailwindcss/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Review by innocenzi — CHANGES_REQUESTED** [review]
Pretty nice work! Please update the description to abide to the [guidelines](https://github.com/aniftyco/awesome-tailwindcss/blob/master/.github/CONTRIBUTING.md#formats-naming-conventions-and-descriptions) and I can merge this :)

**Comment by paul-oms** [comment]
Thanks @innocenzi - updated the description - hope it's ok now? Thanks!

File: README.md [hunk 1]
```diff
@@ -269,7 +269,7 @@
 - 📚 [a17t](https://a17t.miles.land) - Atomic design toolkit built to extend Tailwind CSS.
 - 📚 [tails-ui](https://github.com/knipferrc/tails-ui) - React UI library using Tailwind CSS.
 - 📚 [tails](https://github.com/thedevdojo/tails) - Hand-crafted templates and components using Tailwind CSS.
-- 📁 [OhMySMTP Templates](https://github.com/ohmysmtp/templates) - A set of Transactional HTML Email Templates, built with Tailwind CSS and Maizzle
+- 📁 [OhMySMTP Templates](https://github.com/ohmysmtp/templates) - Set of Transactional HTML Email Templates, built with Maizzle
```
Comment (innocenzi) [inline README.md:272]:
```suggestion
- 📁 [OhMySMTP Templates](https://github.com/ohmysmtp/templates) - Set of transactional HTML email templates, built with Maizzle.
```

**Review by innocenzi — APPROVED** [review]

**Comment by innocenzi** [comment]
Almost, but I updated it myself. Thanks!

**Pull request merged**